04-data-structures-part1.Rmd

```diff
@@ -339,6 +339,17 @@ In this case, we've explicitly told R that "control" should represented by 1, an
 "case" by 2. This designation can be very important for interpreting the
 results of statistical models!
 
+> ## Challenge 5 {.challenge}
+> The following code creates two variables of type factor:
+> ```{r, eval=FALSE}
+> sex <- factor(c("M", "F", "F", "M", "M", "F"))
+> age <- factor(sample(100, size=6))
+>```
+> Try to get character vector of sex values from `sex` factor
+> and numeric vector from `age`.
+> Does using `as.character()` and `as.numeric()` return results you expect?
```

Comment:
I would reword to:

"Does using the function `as.character()` or the function `as.numeric()` return the results you expect?"